dockers/docker-dhcp-relay/cli/show/plugins/show_dhcp_relay.py

```diff
@@ -49,9 +49,8 @@ def __init__(self):
     def get_interface(self):
         """ Get all names of all interfaces in DHCPv6_COUNTER_TABLE """
         vlans = []
-        for key in self.db.keys(self.db.STATE_DB):
-            if DHCPv6_COUNTER_TABLE in key:
-                vlans.append(key[21:])
+        for key in self.db.keys(self.db.STATE_DB, DHCPv6_COUNTER_TABLE + "|*"):
+            vlans.append(key[21:])
```

Comment:
How to explain this magic number?
Can we use define it in __init__ method with calculation? such as `len(self.table_name)`?